File: android/versioned-abis/expoview-abi36_0_0/src/main/java/abi36_0_0/expo/modules/camera/tasks/BarCodeScannerAsyncTaskDelegate.java
package abi36_0_0.expo.modules.camera.tasks;

import abi36_0_0.org.unimodules.interfaces.barcodescanner.BarCodeScannerResult;

public interface BarCodeScannerAsyncTaskDelegate {
  void onBarCodeScanned(BarCodeScannerResult barCode);
  void onBarCodeScanningTaskCompleted();
}

File: android/versioned-abis/expoview-abi47_0_0/src/main/java/abi47_0_0/expo/modules/camera/tasks/BarCodeScannerAsyncTaskDelegate.kt
package abi47_0_0.expo.modules.camera.tasks

import abi47_0_0.expo.modules.interfaces.barcodescanner.BarCodeScannerResult

interface BarCodeScannerAsyncTaskDelegate {
  fun onBarCodeScanned(barCode: BarCodeScannerResult)
  fun onBarCodeScanningTaskCompleted()
}
